src/components/main-page/EventDetails/EventDetails.tsx

```diff
@@ -10,9 +10,24 @@ export function EventDetails() {
       <Heading className={styles.heading} withAccent>
         {t("eventDetails.title")}
       </Heading>
-      <Text className={styles.text}>{t("eventDetails.description1")}</Text>
-      <Text className={styles.text}>{t("eventDetails.description2")}</Text>
-      <Text className={styles.text}>{t("eventDetails.description3")}</Text>
+      <Text className={styles.text}>
+        <Trans
+          i18nKey="eventDetails.description.main"
+          components={{ b: <b /> }}
```

Comment:
It's not a big deal, but maybe we could use `<strong>`  instead of  `<b>` to emphasize meanings of these words

Reply:
Yeah, it's not a big deal, but really good point to emphasize meaning and not styling here - adjusted accordingly in https://github.com/akai-org/akai-code/pull/44/commits/1e73bfcc3a60dfe57d8048d6a537323d18ce4b8e

Reply:
Yeah, it's not a big deal, but really good point to emphasize meaning and not styling here - adjusted accordingly in 1e73bfc